libs/http.liq

```diff
@@ -163,7 +163,10 @@ def harbor.http.static.base(serve,~content_type,~basepath,~headers,~browse,direc
   end
 
   def register(method)
-    serve(method=method,"^#{basepath}",handler(method=method))
+    def handler(~protocol,~data,~headers,uri)
+      handler(method=method, protocol=protocol, data=data, headers=headers, uri)
+    end
```

Comment:
Maybe a good opportunity to cleanup and generalize the use of `%argsof`?

Reply:
I will think of something with records, but for next time!